components/eamxx/CMakeLists.txt

```diff
@@ -66,18 +67,19 @@ if (NOT SCREAM_CIME_BUILD)
     list(REMOVE_ITEM CMAKE_CXX_IMPLICIT_LINK_LIBRARIES "ifport")
   endif()
 
-  # Print the sha of the last commit (useful to double check which version was tested on CDash)
-  execute_process (COMMAND git rev-parse HEAD
-    WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-    OUTPUT_VARIABLE LAST_GIT_COMMIT_SHA
-    OUTPUT_STRIP_TRAILING_WHITESPACE)
-  set(LAST_GIT_COMMIT_SHA ${LAST_GIT_COMMIT_SHA} CACHE STRING "The sha of the last git commit.")
-  message(STATUS "The sha of the last commit is ${LAST_GIT_COMMIT_SHA}")
 else()
   # Ensure our languages are all enabled
   enable_language(C CXX Fortran)
 endif()
 
+# Print the sha of the last commit (useful to double check which version was tested on CDash)
```

Comment:
I don't recall why we moved this out of the if statement above. Not that it matters much, but maybe sha info should be printed at the top of E3SM cmake machinery (easy to find).